diff -urN aa/gcalctool-4.3.44/configure.in gcalctool-4.3.44/configure.in
--- aa/gcalctool-4.3.44/configure.in	2004-02-09 16:56:42.000000000 +0100
+++ gcalctool-4.3.44/configure.in	2004-02-13 23:13:17.683183568 +0100
@@ -74,7 +74,7 @@
 
 dnl Add the languages which your application supports here.
 AM_PROG_LIBTOOL
-ALL_LINGUAS="am ar az be bg ca cs cy da de el en_CA en_GB es eu fa fi fr ga he hi hr hu id it ja ko lt mk ml mn ms nl no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th uk vi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg ca cs cy da de el en_CA en_GB es eu fa fi fr ga he hi hr hu id it ja ko lt mk ml mn ms nb nl pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th uk vi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 dnl Use -Wall if we have gcc.
