--- gcalctool-4.4.18/gcalctool.desktop.in.orig	2004-09-16 23:38:43.558160072 +0200
+++ gcalctool-4.4.18/gcalctool.desktop.in	2004-09-16 23:39:11.340936448 +0200
@@ -7,7 +7,7 @@
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;Application;Utility;
+Categories=GTK;GNOME;Utility;Calculator;
 X-GNOME-DocPath=gcalctool/gcalctool.xml
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gcalctool
