diff -urN gcalctool-4.4.8.orig/configure.in gcalctool-4.4.8/configure.in
--- gcalctool-4.4.8.orig/configure.in	2004-06-08 16:53:24.592100416 +0200
+++ gcalctool-4.4.8/configure.in	2004-06-08 16:53:43.191272912 +0200
@@ -74,7 +74,7 @@
 
 dnl Add the languages which your application supports here.
 AM_PROG_LIBTOOL
-ALL_LINGUAS="am ar az be bg ca cs cy da de el en_CA en_GB es eu fa fi fr ga gu he hi hr hu id it ja ko lt mk ml mn ms nl no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk uk vi zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg ca cs cy da de el en_CA en_GB es eu fa fi fr ga gu he hi hr hu id it ja ko lt mk ml mn ms nb nl pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tk uk vi zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 
 dnl Use -Wall if we have gcc.
