Install gci include dir consistent with .pc file Cflags
--- gnome-calculator-46.1/gci/meson.build.orig	2024-05-08 22:25:19.660946469 +0200
+++ gnome-calculator-46.1/gci/meson.build	2024-05-08 22:26:03.847373758 +0200
@@ -92,7 +92,7 @@ libgci = library(GCI_VERSIONED_PROJECT_N
 	install : true,
 	install_dir : [
 		true,
-		join_paths (get_option('includedir'), 'gci-@0@'.format (API_VERSION), 'gci'),
+		join_paths (get_option('includedir'), 'gci-@0@'.format (GCI_API_VERSION), 'gci'),
 		vapidir,
 		true
 	])
