--- gnome-calculator-50.0/meson.build.orig	2026-03-14 21:47:26.000000000 +0100
+++ gnome-calculator-50.0/meson.build	2026-08-09 21:01:36.063913384 +0200
@@ -110,12 +110,6 @@ subdir('gci')
 endif
 
 if get_option('app')
-# Extra scripts
-gnome.post_install(
-  glib_compile_schemas: true,
-  gtk_update_icon_cache: true,
-)
-
 libadwaita = dependency('libadwaita-1', version: '>= 1.8.alpha')
 gtksourceview = dependency('gtksourceview-5', version: '>= 5.3.0')
 subdir('data')
